doc/source/how-to/continuous-integration.rst

```diff
@@ -176,3 +176,354 @@ Workflow examples are provided for checking :ref:`Coding style`,
+  |  Access deployment status                                                           |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **public_repo**                                                                     | |:x:|                  |
+  |  Access public repositories                                                         |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **repo:invite**                                                                     | |:x:|                  |
+  |  Access repository invitations                                                      |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **security_events**                                                                 | |:x:|                  |
+  |  Read and write security events                                                     |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **workflow**                                                                        | |:x:|                  |
+  |  Update GitHub Action workflows                                                     |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **write:packages**                                                                  | |:x:|                  |
+  |  Upload packages to GitHub Package Registry                                         |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **read:packages**                                                                   | |:x:|                  |
+  |  Download packages from GitHub Package Registry                                     |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **delete:packages**                                                                 | |:x:|                  |
+  |  Delete packages from GitHub Package Registry                                       |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **admin:org**                                                                       | |:x:|                  |
+  |  Full control of orgs and teams, read and write org projects                        |                        |
```

Comment:
Spell out orgs. Maybe Full control of organizations and teams and all read and write permissions.

Reply:
> Spell out orgs. Maybe "Full control of organizations and teams and all read and write permissions."
> Same for the subsequent lines with similar descriptions.

@PipKat this is copy paste from the token page in github that is why I haven't spell out orgs yet. Do you think we really should?